editor/src/uuiui/radix-components.tsx

```diff
@@ -286,6 +293,7 @@ export const RadixSelect = React.memo(
               borderRadius: 6,
               color: 'white',
               boxShadow: UtopiaStyles.shadowStyles.high.boxShadow,
+              zIndex: 3,
```

Comment:
this looks scary

Reply:
was set to `auto` to make sure the dropdown will always be the on top